source/material_model/rheology/strain_dependent.cc

```diff
@@ -650,6 +674,7 @@ namespace aspect
             if (healing_mechanism != no_healing)
               {
                 // Temperature-dependent healing occurs independent of deformation state
+                // or fracture healing mechanism
```

Comment:
This comment doesn't seem to make sense. Maybe:
Temperature-dependent healing or fracture healing occurs independent of deformation state.